torchgeo/trainers/regression.py

```diff
@@ -219,3 +225,129 @@ def configure_optimizers(self) -> Dict[str, Any]:
                 "monitor": "val_loss",
             },
         }
+
+
+class DenseRegressionTask(RegressionTask):
```

Comment:
Is Dense Regression a name that people know this by? I would call it Pixel-Wise Regression personally.

Reply:
I think it could go either way.

Reply:
I've seen several depth estimation papers refer to it as dense regression but I'm not attached to the name.

Reply:
I wonder if we can do the equivalent of a google trends search but for google scholar. I feel like @calebrob6 showed me a plot like that once.

Reply:
After googling both `dense regression remote sensing` and `pixelwise regression remote sensing` it seems that more RS papers refer to it as `pixelwise`. I'm biased towards `dense` because I've seen many monocular depth estimation papers for natural images refer to it as `dense`. I think the RS community tends towards `pixelwise` so we will go with that.